### ISSUE TITLE
tooltip not well  stylized when use the full last version of jquery ui [css or js conflict ?]

### ISSUE THREAD

**Issue by jipexu** [twbs/bootstrap#20913]
with this contexte (os x safari)
bs alpha 4 
jquery 3.1.1
jquery ui 1.12.1
tether
==> the tooltip are  not well  stylized 

if i use the same contexte with a jquery ui 1.12.1 build without the tooltip widget
==>  the tooltip are ok
So it's seems this conflict belong to this widget...

somebody have already experimented this ?...
is it a css or js conflict ?

if is not good place for this please move this post elsewhere

Thanks

**Comment by mdo** [comment]
Probably a conflict with jQuery UI, which we don't integrate with.

**Comment by danyhiol** [comment]
Actually, there is a conflict with bootstrap tooltip. Both (jQuery UI and Bootstrap tooltip) uses the same namespace "tooltip()" for triggering tooltip. I'm also experimenting this issue. Was about to post an issue before I found this post. I've tried everything proposed on the web to solved this ( `$.widget.bridge('uitooltip', $.ui.tooltip); - var bootstrapTooltip = $.fn.tooltip.noConflict();  $.fn.bstooltip = bootstrapTooltip;` ), but none has worked. 
I will be glad if anyone can help on this. I'm using it with meteor with `mizzao:jquery-ui@1.11.4`
